101-managed-application/azuredeploy.json

```diff
@@ -0,0 +1,81 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "applicationName": {
+            "type": "string",
+            "metadata": {
+                "description": "Managed application name"
+            }
+        },
+        "storageAccountNamePrefix": {
+            "type": "string",
```

Comment:
add a defaultValue with uniqueString() to make it globally unique

Reply:
I switched to the storageSelector and this become not a prefix but actual storage account name.